Gruntfile.js

```diff
@@ -19,9 +19,53 @@ module.exports = function (grunt) {
                 },
                 src: ['test/**/*.test.js']
             }
+        },
+        copy: {
```

Comment:
Yes, a local copy is expected. Let's revisit that once this PR is closer to being finished and we can run it live to see what feels best.